src/db/Place/place.ts

```diff
@@ -6,16 +6,30 @@ import { addRating } from "../Rating/rating";
 import { DbOperationError } from "../../errorClasses";
 
 // generate a new rating object with the given id and random ratings from 0 to 5 in increments of 0.5
+// as well as random boolean responses for yes/no questions
 // internal function, no need to export
 const generateNewRating = (placeId: string, userId: ObjectId): Rating => {
+	//TODO: update design
+	const randomNumericRating = () => Math.floor(Math.random() * 12) / 2;
```

Comment:
Are we allowing 0s?
I don't remember what we decided

Reply:
idk if we still wanna go with this, but that was the original idea

Reply:
as discussed, please update this logic so that we only have numbers 1 - 5, plus an equal change for null